pyo3-build-config/src/impl_.rs

```diff
@@ -691,6 +691,11 @@ pub struct PythonVersion {
 }
 
 impl PythonVersion {
+    pub const PY313: Self = PythonVersion {
+        major: 3,
+        minor: 13,
+    };
+    #[allow(dead_code)]
```

Comment:
this is leftover from a mistaken refactoring that removed the only internal use of this outside tests and should be deleted